File: Project/MyProject/MyProject/Bak/Being/PlayerOtherChildMovement.cs
namespace SDK.Lib
{
    public class PlayerOtherChildMovement : PlayerChildMovement
    {
        public PlayerOtherChildMovement(SceneEntityBase entity)
            : base(entity)
        {

        }

        override public void onTick(float delta, TickMode tickMode)
        {
            base.onTick(delta, tickMode);
        }
    }
}

File: Project/MyProject/MyProject/Source/MyProject/Private/BasicFrame/Module/Common/Scene/Being/PlayerOtherChildMovement.cpp
#include "MyProject.h"
#include "PlayerOtherChildMovement.h"
#include "MClassFactory.h"

MY_BEGIN_NAMESPACE(MyNS)

PlayerOtherChildMovement::PlayerOtherChildMovement(SceneEntityBase* entity)
	: base(entity)
{

}

void PlayerOtherChildMovement::onTick(float delta, TickMode tickMode)
{
	Super::onTick(delta, tickMode);
}

MY_END_NAMESPACE